go.mod

```diff
@@ -208,7 +209,7 @@ require (
 	github.com/rs/cors v1.9.0 // indirect
 	github.com/rs/xid v1.4.0 // indirect
 	github.com/sean-/seed v0.0.0-20170313163322-e2103e2c3529 // indirect
-	github.com/sercand/kuberesolver v2.4.0+incompatible // indirect
+	github.com/sercand/kuberesolver/v4 v4.0.0 // indirect
```

Comment:
I think it's worth calling out this change, which was required by the gRPC update, just in case it works differently.

Reply:
Also added to changelog.